library/src/main/kotlin/com/elbekd/bot/types/update.kt

```diff
@@ -22,75 +22,90 @@ internal class UpdateResponse(
     @SerialName("chat_join_request") val chatJoinRequest: ChatJoinRequest? = null,
 )
 
+@Serializable
```

Comment:
these classes are not serializable. fixed in 2.1.5 version

Reply:
Yeah, thanks, you've fixed two problems I had mentioned!

However, there's one more little bug in this PR I'd like to fix (changed title and description)